docs/getting-started/installing/installing-core-linux.md

```diff
@@ -122,17 +74,50 @@ The next thing you will need is a `project.json` file that will outline the depe
 
 ## Run your App
 
-You need to restore packages for your app, based on your project.json, with `dnu restore`.
+You need to restore packages for your app, based on your project.json, with `dotnet restore`.
 
 ```console
-dnu restore
-dnx run
+dotnet restore
+dotnet run
 
 Hello, Linux
-Love from CoreCLR.
+Love from .NET Core.
+
+```
+
+## Compile your application
 
+Running from source is great for rapid prototyping and trying out things. However, in due time you will want to actually compile your application to get increase in speed and similar benefits. In order to do that, we will use `dotnet compile` command that will produce a runnable executable for our Hello World app.
+
+While you're still in the application's directory, type
+
+    dotnet compile
+    
+This will produce a `bin` directory in your directory. The structure of the drop path is `./bin/[configuration]/[framework]/`. Configuration refers to either *Release* or *Debug*, while framework is essentially a framework ID (i.e. dnxcore50). Inside this directory there will be several files, the most important of which is the binary that will have the same name as your application. Running this will give us the message we saw in the previous example. 
+
+**Note**: this binary requires a shared runtime to be installed on the machine. If you wish to create a self-contained application that includes the runtime and that you can just copy over to another machine, you will need to use `dotnet publish` command or compile your application 
```

Comment:
Is there going to be documentation for Shared runtimes, servicing runtimes, etc? We should consider linking it from here to avoid the reader the headache of finding it and understanding the semantics.